src/fsharp/Fsc/Fsc.fsproj

```diff
@@ -30,23 +31,9 @@
       <InProject>false</InProject>
     </FilesToLocalize>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="InternalsVisibleTo.fs" />
-    <Compile Include="..\..\assemblyinfo\assemblyinfo.fsc.exe.fs">
-      <Link>Resources/assemblyinfo.fsc.exe.fs</Link>
-    </Compile>
-    <Compile Include="..\fscmain.fs">
-      <Link>fscmain.fs</Link>
-    </Compile>
-    <None Include="fsc.exe.config">
-      <Link>fsc.exe.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\windows\default.win32manifest">
-      <Link>default.win32manifest</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+
+  <Import Project="Fsc.Sources.proj" />
```

Comment:
That's something I'm willing to live with if it means punting all project system issues in the future.  The next phase of the .NET project system is to support .NET Framework projects as well, so it's likely that some legacy projects (should they be opened in VSvNext) will "just work".